WHITEPAPER.md

```diff
@@ -82,7 +82,13 @@ Ideally, a `node` with higher `throughput` will be rewarded more highly than one
 
 A client in the system that receives multiple `bridge responses`, each with their own price per MB, can choose whichever one meets the `Quality of Service` that it seeks. For low-latency applications, a client may choose to pay more, whereas for simple text based network content, a lower price may be more desirable.
 
-# 6. 
+# 6. Legal Implications of Relay Nodes
```

Comment:
I don't think this should be in the white paper. Can you create a new doc called `TOR.md` and put it in there in a `legal` section?

Reply:
I will do it right now